src/types.ts

```diff
@@ -6,5 +6,9 @@ export type AResult = {
   robust: number;
   robustDeviation: number;
   highLimit: number;
-  lowLimit: number;  
-};
\ No newline at end of file
+  lowLimit: number;
+};
+export type ReferenceResult = {
+  max: number;
+  formula: string;
+} | null;
```

Comment:
No need to add `| null` here, we can move `| null` to the function itself.

Reply:
We can't put null value without null type.

![Screen Shot 2022-06-21 at 13 59 17](https://user-images.githubusercontent.com/18561981/174784029-db46bdef-688c-4a7e-8196-bda07c1d9d4f.png)

![Screen Shot 2022-06-21 at 14 01 42](https://user-images.githubusercontent.com/18561981/174784469-358d2d0f-fd3b-4450-b958-ed996b85915f.png)

Reply:
You can add it here: `): ReferenceResult | null {}` in reference-value.ts

Reply:
Let's fix this as well please